examples/compute_instance/disk_snapshot/main.tf

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "google" {
+  version = "~> 3.0"
+}
+
+# Building the list of disk names in the required format.
+# Usually you would build this list from the outputs of the compute_instance module
```

Comment:
Why can't we pull this as an output from the instance module in this case?

Reply:
Oh yeah, it was still stuck in my mind that `compute_instance` module does not outputs the required details, but forgot I've changed its outputs. Done